File: java-to-kotlin-starter-guide/src/main/java/me/sungbin/lec01/Lec01Main.java
package me.sungbin.lec01;

import java.util.Arrays;
import java.util.List;

public class Lec01Main {
    public static void main(String[] args) {
        long number1 = 10L;
        final long number2 = 10L;

        Long number3 = 1_000L;
        Person person = new Person("양성빈");

        final List<Integer> numbers = Arrays.asList(1, 2);
        numbers.add(3);
    }
}

File: java-to-kotlin-starter-guide-master/src/main/kotlin/me/sungbin/lec01/Lec01Main.kt
package me.sungbin.lec01

fun main() {
    var number1: Long = 10L // 타입 생략 가능
    number1 = 5L

    val number2: Long = 10L // 타입 생략 가능
//    number2 = 5L

    var number3: Long
    number3 = 10L
    println(number3)

    val number4: Long
    number4 = 10L
    println(number4)

    var number5 = 10L
    var number6 = 1_000L

    var number7: Long? = 1_000L
    number7 = null

    val person = Person("양성빈")
}